session.go

```diff
@@ -301,6 +302,14 @@ func (s *Session) Call(ctx context.Context, op interface{}, resp interface{}) er
 		return err
 	}
 
+	if len(reply.Errors) > 0 {
+		err := errors.New("rpc call returned errors")
```

Comment:
I am not sure this error helps at all.  The specific RPC errors should be good enough